docs/sources/http_api/user.md

```diff
@@ -69,6 +69,38 @@ parent = "http_api"
       "isGrafanaAdmin": true
     }
 
+## Get single user by Username(login) or Email
+
+    `GET /api/users/loginoremail/:loginOrEmail`
+
+    **Example Request using the email option**:
+
+        GET /api/loginoremail/user@mygraf.com HTTP/1.1
```

Comment:
the example url looks wrong here. 

I think a better api url / action would be   `/api/users/lookup?loginOrEmail=mylogin`   that is use a query parameter, and a `lookup` URL resource